.ci/Jenkinsfile

```diff
@@ -271,7 +271,16 @@ def doSlackSend(String color) {
 
   def message = "*[${testsSuites}]* Build `${buildStatus}`: ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.RUN_DISPLAY_URL}|Open>)."
 
-  slackSend(channel: "#${env.SLACK_CHANNEL}", color: "${color}", message: "${message}",tokenCredentialId: 'jenkins-slack-integration-token')
+  def channels = "${env.SLACK_CHANNEL}"
+  if (channels?.trim()) {
+    channels = "observablt-bots"
+  }
+
+  channels.split(',').each { channel ->
+    channel = channel.trim()
+    log(level: 'DEBUG', text: "Sending '${buildStatus}' slack message to '${channel}'")  
+    slackSend(channel: "#${channel}", color: "${color}", message: "${message}",tokenCredentialId: 'jenkins-slack-integration-token')
+  }
```

Comment:
After https://github.com/elastic/apm-pipeline-library/pull/803, the library will support multiple channels in the notify build result